wordpress-mobile/WordPress-Android
# Stats revampv2 empty states for some cards

This PR updates empty states for **Most Popular Time** and **Latest Post Summary** and **Views & Visitors** cards

| Most Popular Time | Latest Post Summary | Views & Visitors
|---|---|---|
|![Screenshot_20220524_160239](https://user-images.githubusercontent.com/990349/170021697-bd830035-dfa8-4af0-9b02-be358e948483.png) | ![Screenshot_20220524_163348](https://user-images.githubusercontent.com/990349/170021753-3469a2e4-ff6c-4043-a093-8cf6a90f6052.png)| ![Screenshot_20220526_162239](https://user-images.githubusercontent.com/990349/170429703-b022bbed-79e1-4a87-91c6-28da1c5b9418.png)|



Fixes #16617 

To test:

Setup:

- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find StatsRevampV2FeatureConfig under Features in development enable it and restart app

Test 1:
- Launch app
- Choose a site that has no data (Preferably a newly minted site)
- Go to stats either using quick links or menu 
- Make sure Stats is on **Insights** tab if not switch to it.
- Make sure **Most Popular Time** card is displaying, if not use Stats Management to add it (scroll down if necessary)
- Ensure that it display empty data state as shown above in the first column

Test 2:
- Make sure **Latest Post Summary** card is displaying, if not use Stats Management to add it (scroll down if necessary)
- Ensure that it display empty data state as shown above in the second column

Test 3:
- Make sure **Views and Visitors** card is displaying, if not use Stats Management to add it (scroll down if necessary)
- Ensure that it display empty data state as shown above in the third column
- Click on the icon, check that it takes to top tips page

NOTE:
- Create post is an existing action on **Latest Post Summary** card and is kept as it is along with other actions on that card.


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
Existing and updates tests

PR submission checklist:
